Readme.md

```diff
@@ -79,9 +79,8 @@ Defaults to `rfc6750`.
   to one of these values. If the function returns or resolves to any other value, rejects, or throws,
   an HTTP status of `500` will be sent. `req` is the Fastify request object. If `auth` is a function,
   `keys` will be ignored. If `auth` is not a function or `undefined`, `keys` will be used
-* `addHook`: If `false`, this plugin will not register `onRequest` hook automatically.
-  Instead it provides two decorations: `fastify.verifyBearerAuth` and
-   `fastify.verifyBearerAuthFactory`
+* `addHook`: If `false`, no hook is registered automatically, and instead the `fastify.verifyBearerAuth` and `fastify.verifyBearerAuthFactory` decorators are exposed.
+  If `true` or nullish, defaults to `onRequest`. `onRequest` or `preParsing` can also be used to register the respective hook
```

Comment:
Why would a null-like value be a true value?

Reply:
Whoops, scrubbed in https://github.com/fastify/fastify-bearer-auth/pull/212/commits/341250b3c254462b1af801bb5f3451a5732522db